CombinatorialGames/IGame/Short.lean

```diff
@@ -75,6 +75,26 @@ macro "sgame_wf" : tactic =>
     [Prod.Lex.left, Prod.Lex.right, PSigma.Lex.left, PSigma.Lex.right,
     IsOption.moveLeft, IsOption.moveRight] )
 
+def toStringAux {n : ℕ} (f : Fin n → String) : String :=
```

Comment:
Despite this being an auxiliary definition, I am uncomfortable with the name being `toStringAux` despite it acting on `Fin n -> String` instead of `SGame`.

Reply:
I'll simply inline this into `toString'`.